app/enterprise/2.3.x/vitals/vitals-influx-strategy.md

```diff
@@ -9,45 +9,106 @@ redirect_from:
+### Step 1. Install {{site.base_gateway}}
 
-This guide assumes an existing InfluxDB server or cluster is already installed
-and is accepting write traffic. Production-ready InfluxDB installations should
-be deployed as a separate effort, but for proof-of-concept testing, running a 
-local InfluxDB instance is possible via Docker:
+If you have not installed {{site.base_gateway}}, a Docker installation will work for the purposes of this guide. 
+See [Install {{site.base_gateway}} on Docker](/enterprise/{{page.kong_version}}/deployment/installation/docker)
+for installation instructions.
+
+**When you get to
+[Step 5. Start the gateway with Kong Manager](/enterprise/{{page.kong_version}}/deployment/installation/docker/#start-gateway),
+use the following configuration instead:**
+
+```bash
+$ docker run -d --name kong-ee --network=kong-ee-net \
+  -e "KONG_DATABASE=postgres" \
+  -e "KONG_PG_HOST=kong-ee-database" \
+  -e "KONG_PG_PASSWORD=kong" \
+  -e "KONG_PROXY_ACCESS_LOG=/dev/stdout" \
+  -e "KONG_ADMIN_ACCESS_LOG=/dev/stdout" \
+  -e "KONG_PROXY_ERROR_LOG=/dev/stderr" \
+  -e "KONG_ADMIN_ERROR_LOG=/dev/stderr" \
+  -e "KONG_ADMIN_LISTEN=0.0.0.0:8001" \
+  -e "KONG_ADMIN_GUI_URL=http://<DNSorIP>:8002" \
+  -e "KONG_VITALS_STRATEGY=influxdb" \
+  -e "KONG_VITALS_TSDB_ADDRESS=influxdb:8086" \
+  -p 8000:8000 \
+  -p 8443:8443 \
+  -p 8001:8001 \
+  -p 8444:8444 \
+  -p 8002:8002 \
+  -p 8445:8445 \
+  -p 8003:8003 \
+  -p 8004:8004 \
+  kong-ee
+```
+
+<div class="alert alert-ee blue">
+<strong>Note:</strong> For <code>KONG_ADMIN_GUI_URL</code>, replace <code>&lt;DNSorIP&gt;</code>
+with with the DNS name or IP of the Docker host. <code>KONG_ADMIN_GUI_URL</code>
+<i>should</i> have a protocol, for example, <code>http://</code>.
+</div>
+
+If you already have a {{site.base_gateway}} instance, skip to Step 2.
+
+### Step 2. Deploy a {{site.ee_product_name}} License
+
+You will not be able to access the Kong Vitals functionality without a valid
+{{site.ee_product_name}} license attached to your {{site.base_gateway}} instance.
+
+See [Deploy an Enterprise License](/enterprise/{{page.kong_version}}/deployment/licenses/deploy-license)
+for instructions to apply a license to your {{site.base_gateway}} instance.
+
+If you already have a {{site.ee_product_name}} license attached to your {{site.base_gateway}}
+instance, skip to Step 3.
+
+### Step 3. Start an InfluxDB database
+
+Production-ready InfluxDB installations should be deployed as a separate
+effort, but for proof-of-concept testing, running a local InfluxDB instance
+is possible via Docker:
 
 ```bash
 $ docker run -p 8086:8086 \
-      -v $PWD:/var/lib/influxdb \
-      -e INFLUXDB_DB=kong \
-      influxdb
+  --network=<YOUR_NETWORK_NAME> \
+  --name influxdb \
+  -e INFLUXDB_DB=kong \
+  influxdb:1.8.4-alpine
 ```
 
+<div class="alert alert-warning">
+You <strong>must</strong> use the listed version of
+InfluxDB as the most recent version will <strong>not</strong> work.  
```

Comment:
```suggestion
InfluxDB because the most recent version will <strong>not</strong> work.  
```

Is this always the case?

Reply:
The most recent version InfluxDB 2.0 does not work bc they added a UI and so the way Kong connects to it does not work the usual way. I need to play around and see if I can get 2.0 to work some other way, but for now Jack recommended they stick with v1.8.4 as it is a more stable version anyway.

Reply:
I will update wording to be more clear.